components/card/Card.html

```diff
@@ -0,0 +1,448 @@
+              </li>
+            </a>
+            <a href="../input/Input.html">
+              <li class="sidebar-list-item">
+                <span class="sidebar-icon"><i class="fas fa-edit"></i></span>
+                <span class="sidebar-item-text">Input</span>
+              </li>
+            </a>
+          </ul>
+        </nav>
+
+        <!-- card section -->
+        <div class="example-container">
+          <h1 class="example-title">Cards</h1>
+          <p class="example-desc">
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ut iure
+            debitis sint vitae voluptatem placeat, cupiditate sequi minus
+            laudantium quo.
+          </p>
+
+          <!-- text only card -->
+          <hr class="example-hr" />
+          <h2>Text only card</h2>
+          <p class="example-desc">
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quo illo
+            voluptas assumenda? Inventore, reprehenderit exercitationem?
+          </p>
+
+          <div class="example-showcase">
+            <div class="card">
+              <div class="card-primary">
+                <h1 class="card-primary-title">Card title</h1>
+              </div>
+              <div class="card-secondary">
+                <p>
+                  Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                  Architecto voluptas fugiat omnis eligendi totam facilis cum?
+                </p>
+              </div>
+              <div class="card-actions">
+                <button class="card-actions-btn-filled">Button</button>
+              </div>
+            </div>
+          </div>
+
+          <code>
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%2843%2C43%2C43%2C1%29&t=seti&wt=none&l=htmlmixed&width=800&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522card%2522%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522card-primary%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Ch1%2520class%253D%2522card-primary-title%2522%253ECard%2520title%253C%252Fh1%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522card-secondary%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253ELorem%2520ipsum%2520dolor%2520sit%2520amet%2520consectetur%2520adipisicing%2520elit.%253C%252Fp%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522card-actions%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522card-actions-btn-filled%2522%253EButton%253C%252Fbutton%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+              style="
+                width: 100%;
+                height: 160px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
```

Comment:
You can move your iframe styling to one of the common stylesheet's. It'll help with maintenance and keep and html clean.

Reply:
Ya that would make the code look more clean, thanks.